packages/grid/test/column-rendering.common.js

```diff
@@ -113,6 +113,18 @@ import { flushGrid, getCellContent, getHeaderCellContent, onceResized } from './
       expectBodyCellNotRendered(columns.length - 1);
     });
 
+    it('should not render hidden rows on renderer change', async () => {
+      grid.items = [{ name: 'Item 1' }, { name: 'Item 2' }];
+      grid._getRenderedRows()[1].hidden = true;
```

Comment:
Reduce the grid's item count rather than manually hiding a row element. This will make virtualizer hide the second row.

```suggestion
      grid.items = [{ name: 'Item 1' }];
```